srtcore/logging.h

```diff
@@ -381,7 +382,7 @@ struct LogDispatcher::Proxy
             buf[len-1] = '\0';
         }
 
-        os << buf;
+        os.write(buf, len);
```

Comment:
This is done to bypass any formatting features of ostringstream (idem the next one).